### DISCUSSION TITLE
Request for new components, improvements for existing components and more detailed documents.

### DISCUSSION THREAD

**Discussion by HappyCodingCoding** [saadeghi/daisyui#2717] [Ideas / feature requests]
daisyUI is awesome, wonderful and vibrant but is lacking some components compared to other UI libraries like Flowbite and Preline. And there is much space left to be improved in some existing components like the drawer component lacking features of opening up at the top and the bottom. And the document is less detailed compared to other fellows. I know this is not easy and takes time to build. But there are my opinions only and really hope daisyUI would get better and become the best and comprehensive UI library within the Tailwind CSS ecosystem.

**Comment by saadeghi** [comment]
> document is less detailed compared to other fellows

Can you specify which details or information are missing from the docs?

**Comment by HappyCodingCoding** [comment]
For example, the doc of the model components in Preline and Flowbite is more detailed than in daisyUI. Perhaps the key factor effecting the detail of a component' documentation is how feature-rich the component is.
